### ISSUE TITLE
Vulnerable Dependencies

### ISSUE THREAD

**Issue by trevor-gullstad** [sinonjs/sinon#2383]
**Describe the bug**
There are vulnerable packages in `package.json` and `Gemfile.lock`

**To Reproduce**

1. Run `npm install`
2. Run `npm audit`
```
up to date, audited 918 packages in 3s

8 vulnerabilities (4 low, 4 moderate)

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```
```
# npm audit report

glob-parent  <5.1.2
Severity: moderate
Regular expression denial of service - https://npmjs.com/advisories/1751
fix available via `npm audit fix --force`
Will install mochify@8.0.0, which is a breaking change
node_modules/glob-parent
node_modules/watchify/node_modules/glob-parent
  chokidar  1.0.0-rc1 - 2.1.8
  Depends on vulnerable versions of glob-parent
  node_modules/watchify/node_modules/chokidar
    watchify  3.0.0 - 3.11.1
    Depends on vulnerable versions of chokidar
    node_modules/watchify
      mochify  >=2.0.0
      Depends on vulnerable versions of mocaccino
      Depends on vulnerable versions of mocha
      Depends on vulnerable versions of watchify
      node_modules/mochify

minimist  <0.2.1 || >=1.0.0 <1.2.3
Prototype Pollution - https://npmjs.com/advisories/1179
fix available via `npm audit fix --force`
Will install mochify@8.0.0, which is a breaking change
node_modules/mkdirp/node_modules/minimist
  mkdirp  0.4.1 - 0.5.1
  Depends on vulnerable versions of minimist
  node_modules/mkdirp
    mocha  1.21.5 - 6.2.2 || 7.0.0-esm1 - 7.1.0
    Depends on vulnerable versions of mkdirp
    node_modules/mocaccino/node_modules/mocha
    node_modules/mochify/node_modules/mocha
      mocaccino  1.6.0 - 4.1.2
      Depends on vulnerable versions of mocha
      node_modules/mocaccino
        mochify  >=2.0.0
        Depends on vulnerable versions of mocaccino
        Depends on vulnerable versions of mocha
        Depends on vulnerable versions of watchify
        node_modules/mochify

8 vulnerabilities (4 low, 4 moderate)

To address all issues (including breaking changes), run:
  npm audit fix --force
```

1. `sudo gem install bundler-audit`
2. Run `bundle-audit`
```
Name: kramdown
Version: 2.3.0
CVE: CVE-2021-28834
GHSA: GHSA-52p9-v744-mwjj
Criticality: Unknown
URL: https://github.com/advisories/GHSA-52p9-v744-mwjj
Title: Remote code execution in Kramdown
Solution: upgrade to >= 2.3.1

Vulnerabilities found!
```

**Expected behavior**
Dependencies should be upgraded to mitigate vulnerabilities.

**Screenshots**
None

**Context (please complete the following information):**

- Library version: 11.1.1 (latest)
- Environment: macOS
- Example URL: 
- Other libraries you are using: 
  - npm 7.17.0
  - https://github.com/rubysec/bundler-audit

**Additional context**

**Comment by fatso83** [comment]
Thanks for reporting! Not totally sure this is worth spending time on. All of the affected libs are _development_ dependencies, meaning they do not affect any consumers, but I sometimes just do it to get rid of warnings nonetheless :)

**Comment by trevor-gullstad** [comment]
Thanks @fatso83 I'll look out for the next release!
As a follow up can you tell me what the Ruby Gem files are used for?

**Comment by fatso83** [comment]
@trevor-gullstad We use Jekyll to generate our static sinonjs.org site. That is a Ruby project :)

**Comment by fatso83** [comment]
I ran `npm audit` now and all remaining issues are down to `mochify`, which would be solved by #395, if someone could spend time getting that over the finish line.
